### ISSUE TITLE
running npm run setup returns 'SyntaxError'

### ISSUE THREAD

**Issue by Devhound** [strapi/strapi#1093]
**Informations**
- **Node.js version**: v9.11.1
- **npm version**: 5.6.0
- **Strapi version**:  3.0.0-alpha.12.0.1
- **Database**: mongodb
- **Operating system**: ubuntu 16.0.4

**What is the current behavior?**

ran npm run setup after changin admin config. Got error message 
```
SyntaxError: /home/devion/ttu-strapi/plugins/users-permissions/services/UsersPermissions.js: Unexpected token (34:12)
  32 |             controller,
  33 |             action: action.toLowerCase(),
> 34 |             ...params.permissions[type].controllers[controller][action]
     |             ^
  35 |           }));
  36 |         });
  37 |       });
```

**Steps to reproduce the problem**
Followed this tutorial 
https://blog.strapi.io/how-to-deploy-a-strapi-application/

---

- [ x] I'm sure that this issue hasn't already been referenced

**Comment by Devhound** [comment]
I'm also getting 401 errors when trying to create new content types. This doesn't happen locally but on a ubuntu 16.0.4 server it's a different story. I tried to change my admin config to the ip instead of localhost but when I ran npm run setup I got the above error message. I followed the docs and blog post I linked above.

**Comment by nathanmarks** [comment]
I'm seeing the same issue. Happens during the `build` task in admin

**Comment by soupette** [comment]
This issue has been fixed already but we haven't released it #1033 . The patch should be available today.

**Comment by soupette** [comment]
You need to remove the `analytics` plugin then it should be working with the latest release
